packages/babel-plugin-extract-messages/test/fixtures/js-message-descriptor.js

```diff
@@ -1,11 +1,7 @@
-/*i18n*/'Message'
```

Comment:
i think original author's ideawas to test extraction from StringLiteral, but actually this code is a directive prologue according to ecma languge grammar. It would be a StringLiteral only when it assigned to the variable or used sometging else. 

So i deleted it from here and created a specific `it` in the suite